<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsMaskerToolUi</class>
 <widget class="QWidget" name="xToolsMaskerToolUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Masker</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Mask code</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="xToolsSpinBox" name="spinBoxMaskCode">
        <property name="maximum">
         <number>256</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="xToolsCheckBox" name="checkBoxEnable">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">xToolsCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>xToolsSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">xToolsSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
